python/main.py

```diff
@@ -55,27 +60,46 @@ async def lifespan(app: FastAPI):
 class HelloResponse(BaseModel):
     message: str
 
+@app.get("/items") 
+def get_items():
+    with open('items.json', 'r') as json_open:
+        json_load = json.load(json_open)
+    return json_load
 
-@app.get("/", response_model=HelloResponse)
-def hello():
-    return HelloResponse(**{"message": "Hello, world!"})
-
+@app.get("/items/{item_id}") 
+def get_items_by_id(item_id: int):
+    with open('items.json', 'r') as json_open:
+        json_load = json.load(json_open)
+    item = json_load["items"][item_id]
+    return item
 
 class AddItemResponse(BaseModel):
     message: str
 
-
 # add_item is a handler to add a new item for POST /items .
 @app.post("/items", response_model=AddItemResponse)
 def add_item(
     name: str = Form(...),
+    category: str = Form(...),
+    image: UploadFile = File(...),
     db: sqlite3.Connection = Depends(get_db),
 ):
     if not name:
         raise HTTPException(status_code=400, detail="name is required")
+    if not category:
+        raise HTTPException(status_code=400, detail="category is required")
```

Comment:
We should validate `image` similarly as well, as well the extension of the image